CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ ENDIF(CMAKE_VERSION VERSION_GREATER 3.0.2)
 
 PROJECT(opencog)
 
+FIND_PACKAGE(Boost 1.71 REQUIRED date_time program_options thread filesystem system unit_test_framework)
+IF(${Boost_FOUND})
+  INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIR})
+ENDIF()
```

Comment:
Why did you make this change? It's causing unit tests to fail, and does not appear to be needed for anything...

Reply:
I couldn't compile without this On Ubuntu 20.04. I can add it manually later for my selft later